### DISCUSSION TITLE
How to add subpages in Gh pages?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23233] [Other Feature Feedback, Questions, & Ideas]
<p>I have my page working: <a href="https://snvmk.github.io" rel="noopener nofollow ugc">link</a>. I wanna add subpage: <a href="http://snvmk.github.io/snvmdb" rel="noopener nofollow ugc">snvmk.github.io/snvmdb</a> , but i dont know how. I added snvmdb folder to page repo, but it isn`t working. Can you guys explain me how to do this?</p>

**Comment by AlfredoRamos** [comment, marked as answer]
<p>You can create new pages or new posts following these guides:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://jekyllrb.com/favicon.ico" class="site-icon" width="32" height="32">
      <a href="https://jekyllrb.com/docs/pages/" target="_blank" rel="noopener nofollow ugc" title="01:53AM - 03 January 2021">Jekyll • Simple, blog-aware, static sites – 3 Jan 21</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/4165935/181075679-f737b962-d184-417a-a394-41a0e12c5c0e.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_10x10.png"></div>

<h3><a href="https://jekyllrb.com/docs/pages/" target="_blank" rel="noopener nofollow ugc">Pages</a></h3>

<p>Pages are the most basic building block for content. They’re useful for standalone content (content which is not date based or is not a group of content such as staff members or recipes).</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://jekyllrb.com/favicon.ico" class="site-icon" width="32" height="32">
      <a href="https://jekyllrb.com/docs/posts/" target="_blank" rel="noopener nofollow ugc" title="01:53AM - 03 January 2021">Jekyll • Simple, blog-aware, static sites – 3 Jan 21</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/4165935/181075680-a1cb1298-c378-411f-baac-6ea02f83eeb9.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_10x10.png"></div>

<h3><a href="https://jekyllrb.com/docs/posts/" target="_blank" rel="noopener nofollow ugc">Posts</a></h3>

<p>Blogging is baked into Jekyll. You write blog posts as text files and Jekyll provides everything you need to turn it into a blog.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://jekyllrb.com/favicon.ico" class="site-icon" width="32" height="32">
      <a href="https://jekyllrb.com/docs/permalinks/" target="_blank" rel="noopener nofollow ugc" title="01:53AM - 03 January 2021">Jekyll • Simple, blog-aware, static sites – 3 Jan 21</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/4165935/181075682-a1abeedc-631a-41f2-a124-b4e313c58de6.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_10x10.png"></div>

<h3><a href="https://jekyllrb.com/docs/permalinks/" target="_blank" rel="noopener nofollow ugc">Permalinks</a></h3>

<p>Permalinks are the output path for your pages, posts, or collections. They allow you to structure the directories of your source code different from the directories in your output.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
